app/views/admin_user/edit.html.erb

```diff
@@ -106,11 +110,14 @@
           This is disabled for banned users as the email address must be
           preserved to enforce the ban. First remove the ban text and then close
           and anonymise the account.
+
+          This is disabled for users whose accounts have not been closed.
+          To erase a user, first close the account.
         </p>
       </div>
 
       <%= form_tag admin_users_account_erasing_index_path(user_id: @admin_user.id), class: 'span3 form form-inline' do %>
-        <% disabled = @admin_user.banned? || @admin_user.is_pro? %>
+        <% disabled = !@admin_user.closed? || @admin_user.is_pro? %>
```

Comment:
Sorry. Not quite what I asked for here: https://github.com/mysociety/alaveteli/pull/7727#discussion_r1195040170

Banned users and closed users aren't the same thing. We should retain disabling this form for banned users too.